includes/discovery/toner.inc.php

```diff
@@ -29,7 +29,16 @@
                     if ($descr != '') {
                         $current  = snmp_get($device, $toner_oid, '-Oqv');
                         $capacity = snmp_get($device, $capacity_oid, '-Oqv');
-                        $current  = ($current / $capacity * 100);
+                        //fix for ricoh devices returning garbage
```

Comment:
Worth wrapping this in `if ($os == 'richoh') {` to limit it's risk for other printers.